docs/content/0.Config/5.Automate/2.Supported_Vehicles.md

```diff
@@ -0,0 +1,14 @@
+# Supported Vehicles
+For vehicles listed below a WiCAN vehicle profiles exists:
```

Comment:
We really need a comment, or just regular text here saying this file is auto-generated, so people don't try to update it manually

Reply:
Done